Update configuration with docker page - Closes 1706 (#1756)
LiskHQ/lisk-service: docs/antora/modules/ROOT/pages/configuration/docker.adoc

= Configuration with Docker
Mona Bärenfänger <mona@lightcurve.io> Muhammad Talha <muhammad.talha@lightcurve.io>
:description: Describes how to configure Lisk Service with Docker.
:toc:
:imagesdir: ../assets/images
:page-previous: /lisk-service/setup/docker.html
:page-previous-title: Installation with Docker
:page-next: /lisk-service/management/docker.html
:page-next-title: Docker commands
:url_references_config: configuration/index.adoc
:url_github_env: https://github.com/LiskHQ/lisk-service/blob/development/docker/example.env
:url_docker_commands: management/docker.adoc

On this page, we discuss how to customize the docker installation of Lisk Service.

== Configuring Lisk Service

> The Docker installation of Lisk Service takes all of its configuration from an `example.env` file, which is located in the `lisk-service` directory.

Reviewer: This statement is incorrect. The `docker/example.env` file is only for reference and specifies all the available configurable environment variables and already specifies the default values.

For the user to use it, they'll need to copy this into a `.env` file and place it next to the `docker-compose.yaml` file in the root directory of Lisk Service.

> To customize the docker installation, copy the example configuration file to any `.env` file.

Reviewer: As above. It specifically has to be a `.env` file.

:page-next-title: Docker commands

:url_references_config: configuration/index.adoc
:url_github_env: https://github.com/LiskHQ/lisk-service/blob/development/docker/example.env
:url_github: https://github.com/LiskHQ/lisk-service
:url_docker_commands: management/docker.adoc

On this page, we discuss how to customize the Docker installation of Lisk Service.

== Configuring Lisk Service
The {url_github}[Lisk Service] provides all possible Docker related configurations and their default values in an `example.env` file.

TIP: The default {url_github_env}[example.env] file can be seen on the GitHub repository.

> To customize the Docker installation, copy the example configuration file to a `.env` file in the root of the `lisk-service` directory, where we have the `docker-compose.yml` file.

Reviewer 2: imho, I would suggest we slightly reword this sentence as follows:
To customize the Docker installation, copy the example configuration file to a `.env` file in the root of the `lisk-service` directory, where the `docker-compose.yml` file is located.
```

> To customize the Docker installation, copy the example configuration file to a `.env` file in the root of the `lisk-service` directory, where we have the `docker-compose.yml` file.

Reviewer: ```suggestion
To customize the Docker installation, copy the example configuration file to a `.env` file in the root of the `lisk-service` directory, where the `docker-compose.yml` file is located.
```
